bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -258,22 +274,6 @@ spec:
                   fi
                   sleep 2
                   done
-
```

Comment:
Might be a stupid question, but is it necessary to do `set-connection` before executing the ovs-db client commands for upgrade? Seems like the error would be caught after retried and cleanly exited (`exit 1` on L257) in case it failed and we wouldn't proceed with upgrading the db? Or is it only for the nbctl commands that follow this block?